### ISSUE TITLE
address in use error

### ISSUE THREAD

**Issue by spencerlyon2** [JunoLab/Blink.jl#49]
Not sure what to make of this one... I can't tell why we are getting it.

Steps to reproduce:

1. Start julia and load blink
2. Keep that one running and start another julia and load blink again.

Then on the second one this is printed:

```
julia> using Blink
listen: address already in use (EADDRINUSE)

julia>  at socket.jl:687
 in listen at /Users/sglyon/.julia/v0.4/HttpServer/src/HttpServer.jl:207
 in run at /Users/sglyon/.julia/v0.4/HttpServer/src/HttpServer.jl:333
 in run at /Users/sglyon/.julia/v0.4/HttpServer/src/HttpServer.jl:343
 [inlined code] from /Users/sglyon/.julia/v0.4/Lazy/src/macros.jl:265
 in anonymous at task.jl:447
```

See gif below:

![blink_error2](https://cloud.githubusercontent.com/assets/1001948/12361828/512f3a14-bb8e-11e5-8e45-2873a2ae44b3.gif)

**Comment by spencerlyon2** [comment]
This issue is actually more than just annoying.

Any Blink after the first is unreachable...

**Comment by spencerlyon2** [comment]
Fixed by b90cd79b427c2800bcb195dac8d4034cc8dacdd7

Thanks!
